aws/resource_aws_cloudtrail.go

```diff
@@ -116,7 +118,21 @@ func resourceAwsCloudTrailCreate(d *schema.ResourceData, meta interface{}) error
 		input.SnsTopicName = aws.String(v.(string))
 	}
 
-	t, err := conn.CreateTrail(&input)
+	var t *cloudtrail.CreateTrailOutput
+	err := resource.Retry(15*time.Second, func() *resource.RetryError {
+		var err error
+		t, err = conn.CreateTrail(&input)
+		if err != nil {
+			if isAWSErr(err, "InvalidCloudWatchLogsRoleArnException", "Access denied.") {
+				return resource.RetryableError(err)
+			}
+			if isAWSErr(err, "InvalidCloudWatchLogsLogGroupArnException", "Access denied.") {
```

Comment:
I think the two if-s look tiny little bit cleaner (easier to read) compared to 1 if with 2 conditionals which would need to be line-wrapped (b/c long lines aren't easy to read either).